Test/Makefile

```diff
@@ -439,6 +441,44 @@ benchmark_large_message:
 
 #################################FLOAT#########################################
 
+#Test float literal values
+test_float_literal_values:
+	echo "Test float literal values"
+	${CREATE_TEST_DIR}
+	cd ${TEST_DIR} && \
+	${SPIN} -run ${FLOAT_MODEL_DIR}/FloatLiteralValues.pml > FloatLiteralValues.output
+	cd ${TEST_DIR} && \
+	${SPIN} ${FLOAT_MODEL_DIR}/FloatLiteralValues.pml > FloatLiteralValues.sim.output
+	grep -q "State-vector 40 byte, depth reached 2, errors: 0" ${TEST_DIR}/FloatLiteralValues.output
+	grep -q "3 states, stored" ${TEST_DIR}/FloatLiteralValues.output
+	grep -q "0 states, matched" ${TEST_DIR}/FloatLiteralValues.output
+	grep -q "3 transitions (= stored+matched)" ${TEST_DIR}/FloatLiteralValues.output
+	grep -q "0 atomic steps" ${TEST_DIR}/FloatLiteralValues.output
+	grep -q "x = 123.000000" ${TEST_DIR}/FloatLiteralValues.sim.output	
+	grep -q "y = 43.000000" ${TEST_DIR}/FloatLiteralValues.sim.output	
+	grep -q "z = 56.099998" ${TEST_DIR}/FloatLiteralValues.sim.output	
+	grep -q "x = 56.130001" ${TEST_DIR}/FloatLiteralValues.sim.output	
```

Comment:
I think that checking if X == A and then X == B, where A != B looks suspicious. I would prefer to modifiy the prints to uniquely indentify the checked value.

Reply:
I think that your comment is actually about the same as "... printf informed about both the values, and their source..." - as you give me more clue on that I would be happy to fix it your way.

Reply:
ok - hopefully done